communications/github-projects.md

```diff
@@ -0,0 +1,59 @@
+## On Github Projects
+
+Since the [Covid Dash development](https://github.com/EQWorks/flaat-dash/projects), we have been using github projects to keep track of progress, issues, and code-specific discussions due to the fast paced nature of the project. 
+
+Github Projects tool can be used in any other repository. Repos can have multiple projects. Projects can be specific to one extensive task or the whole repo in general - ideally both. Projects that have issues intertwined with different repositories (for instance front-end repo and back end repo) can take advantage of the cross-repo issue referencing. 
+
+<img width="800" alt="column" src=https://user-images.githubusercontent.com/53827690/83157518-b1cf8e00-a0d1-11ea-9b2c-7a4019ae5066.png>
+
+Projects are a to-do list: easy to keep track of progress as it shows the percentage between open issues / in progress issues / completed issues.
```

Comment:
suggestion: can mention 'Projects are shown as a kanban board' so it's easy to keep track of progress...'